week_1_basics_n_setup/homework.md

```diff
@@ -1,3 +1,54 @@
 ## Week 1 Homework
 
-Coming soon
\ No newline at end of file
+In this homework we'll prepare the environment 
+and practice with terraform and SQL
+
+## Question 1. Google Cloud SDK
+
+Install Google Cloud SDK. What's the version you have? 
+
+To get the version, run `gcloud --version`
+
+## Question 2. Terraform 
+
+Install Terraform. What's the version you have?
+
+Run `terraform -version` to get it
+
```

Comment:
Also, output of `terraform apply` to check if their GCP infra was created successfully.